titus_isolate/model/kubernetes_workload.py

```diff
@@ -204,3 +194,43 @@ def __get_serializable_pod(self, pod: V1Pod) -> dict:
 
 def get_workload_from_pod(pod: V1Pod) -> KubernetesWorkload:
     return KubernetesWorkload(pod)
+
+def get_job_annotations_from_pod(pod: V1Pod) -> dict:
+    metadata = pod.metadata
+    job_id = None
+    workload_type = STATIC
+
+    if metadata.annotations.get(ANNOTATION_KEY_POD_SPEC_VERSION) is not None:
+        job_id = metadata.annotations.get(V1_ANNOTATION_KEY_JOB_ID, None)
+        job_type = metadata.annotations.get(V1_ANNOTATION_KEY_JOB_TYPE, None)
+        owner_email = metadata.annotations.get(V1_ANNOTATION_KEY_OWNER_EMAIL, None)
+        workload_type_str = metadata.annotations.get(V1_ANNOTATION_KEY_CPU_BURSTING, None)
```

Comment:
e.g. instead of looking up the annotation, we can just always return "STATIC"

Reply:
OK, great - updated with that change.

Reply:
I still see references to `V1_ANNOTATION_KEY_CPU_BURSTING` annotation.  Maybe you forgot to push?

Reply:
I explicitly added tests to make sure that even if you set `V1_ANNOTATION_KEY_CPU_BURSTING`, the type is set to `STATIC`.

Reply:
I don't understand. Maybe github is lying to me about the state of code, but I see the annotations being read here and
```
workload_type = BURST
```
being set sometimes.  I think we should just drop all reference to cpu bursting, as done here:
https://github.com/Netflix-Skunkworks/titus-isolate/pull/340/files

Reply:
It's fine.  If your changes work, we can merge them then merge the other PR.